doc/cheatsheet.md

```diff
@@ -0,0 +1,255 @@
+% djot cheatsheet
+
+## Basics
+
+| Markup | Result |
+| --- | --- |
+| `_italic_` or `{_italic_}` | *italic* |
+| `*bold*` or `{*bold*}` | **bold** |
+| `` `verbatim/code` `` | `verbatim/code` |
+| `H~2~O` | H<sub>2</sub>O |
+| `20^th^` | 20<sup>th</sup> |
+| `{=highlighted=}` | <mark>highlighted</mark> |
+| `{+insert+}` | <ins>insert</ins> |
+| `{-delete-}` | <del>delete</del> |
+
+Use triple-backticks for fenced verbatim blocks:
+
+~~~~~~
+```
```

Comment:
Would be good to note that the language name can be optionally placed after the backticks.

Reply:
Oh, didn't know that djot supported syntax syntax highlighting! Nice!